doc/ServerNetworkStripingRepair.rst

```diff
@@ -0,0 +1,86 @@
+Server Network Striping Repair 
```

Comment:
Personally, I think this is a very detailed document about CORTX Motr SNS repair.
Should it be moved to cortx-motr repo?

Reply:
Yes, if it is part of Motr? Let me know ad I will move it.

Reply:
It is more related to Motr. But I think it is OK and reasonable to keep it here.
This documents will give a conceptual overview of Motr SNS Repair.